manifests/charts/gateways/istio-egress/values.yaml

```diff
@@ -105,7 +105,10 @@ gateways:
     podAntiAffinityTermLabelSelector: []
 
     # whether to run the gateway in a privileged container
-    runAsRoot: false
+    # At this time, because of several crash related issues, the Istio community strongly recommends
+    # running the gateway as root. See:
+    # https://github.com/istio/istio/issues/27566 and https://github.com/istio/istio/issues/26690
```

Comment:
the mentioned crashes are not present anymore

Reply:
Agree. @houshengbo can we revert the default `runAsRoot: true` parts and try to merge this PR?